.github/workflows/delete-namespace.yml

```diff
@@ -63,12 +63,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: checkout repo
-      uses: actions/checkout@v2
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  #v4.1.1
       with:
         repository: 'dBildungsplattform/spsh-app-deploy'
 
     - name: Set up kubectl and Helm
-      uses: azure/setup-kubectl@v3
+      uses: azure/setup-kubectl@3e0aec4d80787158d308d7b364cb1b702e7feb7f #v4.0.0
```

Comment:
Same for this action: Have you tested everywhere that this newer major version works for us like before?

Reply:
Same response here